OptimizelySDK/Utils/EventTagUtils.cs

```diff
@@ -16,5 +18,65 @@ public static object GetRevenueValue(Dictionary<string, object> eventTags)
 
            return eventTags[REVENUE_EVENT_METRIC_NAME];
         }
+
+        public static object GetNumericValue(Dictionary<string, object> eventTags, ILogger logger  = null)
+        {
+            string debugMessage = string.Empty;
+            bool isCasted = false;
+
+            float refVar = 0;
+
+            if (eventTags == null)
+                debugMessage = "Event tags is undefined.";
+
+            else if (!eventTags.ContainsKey(VALUE_EVENT_METRIC_NAME))
+                debugMessage = "The numeric metric key is not in event tags.";
+
+            else if (eventTags[VALUE_EVENT_METRIC_NAME] == null)
+                debugMessage = "The numeric metric key value is not defined in event tags.";
+
+            else if (eventTags[VALUE_EVENT_METRIC_NAME] is bool)
+                debugMessage = "Provided numeric value is boolean which is an invalid format.";
+
+            else if (!(eventTags[VALUE_EVENT_METRIC_NAME] is int) && !(eventTags[VALUE_EVENT_METRIC_NAME] is string) && !(eventTags[VALUE_EVENT_METRIC_NAME] is float)
+                && !(eventTags[VALUE_EVENT_METRIC_NAME] is decimal)
+                && !(eventTags[VALUE_EVENT_METRIC_NAME] is double)
+                && !(eventTags[VALUE_EVENT_METRIC_NAME] is float))
+                debugMessage = "Numeric metric value is not in integer, float, or string form.";
+
+            else
+            {
+                if (!float.TryParse(eventTags[VALUE_EVENT_METRIC_NAME].ToString(), out refVar))
+                    debugMessage = string.Format("Provided numeric value {0} is in an invalid format.", eventTags[VALUE_EVENT_METRIC_NAME]);
+                else
+                {
+                    if (float.IsInfinity(refVar))
+                        debugMessage = string.Format("Provided numeric value {0} is in an invalid format.", eventTags[VALUE_EVENT_METRIC_NAME]);
+                    else
+                        isCasted = true;
+                }
+            }
+                
+
+            if (logger != null)
+            {
+                if (isCasted)
+                    logger.Log(LogLevel.ERROR, string.Format("The numeric metric value {0} will be sent to results.", refVar));
```

Comment:
This should not be logged as an error. It should be INFO.

Reply:
@msohailhussain -- please address this.

Reply:
Yes, mistakenly I made this change, have fixed it.

Reply:
I still don't see this change:
https://github.com/optimizely/csharp-sdk/blob/phase2_ev_ns_fb_kr/OptimizelySDK/Utils/EventTagUtils.cs

Reply:
I can't commit anything on optimizely repo, I can just send the pull request from this repo. 

https://github.com/msohailhussain/csharp-sdk/tree/phase2_ev_ns_fb